includes/Core/Tags/First_Party_Mode/First_Party_Mode.php

```diff
@@ -85,11 +119,21 @@ public function get_debug_fields() {
 		$settings = $this->first_party_mode_settings->get();
 
 		return array(
-			'first_party_mode_is_enabled' => array(
+			'first_party_mode_is_enabled'               => array(
 				'label' => __( 'First-Party Mode: Enabled', 'google-site-kit' ),
 				'value' => $settings['isEnabled'] ? __( 'Yes', 'google-site-kit' ) : __( 'No', 'google-site-kit' ),
 				'debug' => $settings['isEnabled'] ? 'yes' : 'no',
```

Comment:
To add more weight to the helpers' usefulness, we can use them for `isEnabled` too.